x-pack/test_serverless/README.md

```diff
@@ -0,0 +1,66 @@
+# Kibana Serverless Tests
+
+The tests and helper methods (services, page objects) defined here in
+`x-pack/test_serverless` cover the serverless functionality introduced by the
+ `serverless`, `serverless_observability`, `serverless_search` and
+ `serverless_security` plugins.
+
+## Serverless testing structure and conventions
+
+### Overview
+
+The serverless test structure corresponds to what we have in `x-pack/test` with
+API tests in `api_integration` and UI tests in `functional`, each with their
+set of helper methods and sub-directories for
+- `common` functionality shared across serverless projects (core, shared UX, ...)
+- `observability` project specific functionality
+- `search` project specific functionality
+- `security` project specific functionality
+
+```
+x-pack/test_serverless/
+├─ api_integration
+│  ├─ services
+│  ├─ test_suites
+│  │  ├─ common
+│  │  ├─ observability
+│  │  ├─ search
+│  │  ├─ security
+├─ functional
+│  ├─ page_objects
+│  ├─ services
+│  ├─ test_suites
+│  │  ├─ common
+│  │  ├─ observability
+│  │  ├─ search
+│  │  ├─ security
+```
+
```

Comment:
There will be common services used both in `api_integration` and `functional` tests, right?
E.g. apis wrapper for authentication, testing data archives, or even a custom plugin fro testing.
What do you think about adding for the start:
```
x-pack/test_serverless/
|─ common
     |─ services
     |─ fixtures
....
```